.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
+  - name: Code Question
```

Comment:
Given that there are multiple default categories "General", "Ideas", "Q/A", "Show and tell" I don't think we have to limit this to "Code Questions".  Perhaps "OpenZFS Questions" instead and for the about text "Ask the community for help".  This matches the wording form the "Q/A" section of the Discussions page.

Reply:
I was a bit in doubt weither to change it or not.
But will do for sure, because I would prefer the clearer language.

Reply:
Done.